src/main/java/decodex/storage/Storage.java

```diff
@@ -109,9 +100,15 @@ public void loadRecipesFromDirectory(ModuleManager moduleManager, RecipeManager
      */
     private File[] getAllRecipeFiles() {
         File recipeDirectory = new File(DEFAULT_RECIPE_DIRECTORY);
+
         File[] files = recipeDirectory.listFiles();
+
         File[] recipeFiles = Arrays.stream(files)
                 .filter(file -> file.isFile())
+                .filter(file -> {
+                    String fileName = file.getName();
+                    return fileName.contains(RECIPE_FILE_PREFIX);

```

Comment:
Perhaps `.endsWith()` might cover more cases than `.contains()`

Reply:
Or perhaps use a regex check if we want to be much stricter? Like `^[A-Za-z0-9_]\.txt$`?